docs/samples/llmisvc/dp-ep/qwen3-nvidia/llm-inference-service-dp-ep-multi-deepseek-r1-gpu-nvshmem-deepep-private.yaml

```diff
@@ -0,0 +1,271 @@
+
+spec:
+  model:
+    uri: pvc://llm-test-pvc
+    #    uri: hf://deepseek-ai/DeepSeek-R1-0528
+    name: deepseek-ai/DeepSeek-R1-0528
+  replicas: 1
+  parallelism:
+    data: 32
+    dataLocal: 8
+    expert: true
+    tensor: 1
+  router:
+    scheduler:
+      template:
+        imagePullSecrets:
+          - name: "quay-rhoai"
+        containers:
+          - name: main
+            image: quay.io/rhoai/odh-llm-d-inference-scheduler-rhel9:rhoai-2.25
+    route: { }
+    gateway: { }
+  template:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/hostname
+                  operator: In
+                  values:
+                    - pokprod-b93r38s0
+                    - pokprod-b93r38s1
+                    - pokprod-b93r39s1
+                    - pokprod-b93r38s2
+                    - pokprod-b93r38s3
+    imagePullSecrets:
+      - name: "quay-registry-wseaton"
+    serviceAccountName: hfsa
+    containers:
+      - name: main
+        image: quay.io/aipcc/rhaiis/cuda-ubi9:3.2.2-1757699034
+        env:
+          - name: VLLM_LOGGING_LEVEL
+            value: INFO
+          - name: KSERVE_INFER_ROCE
+            value: "true"
+          - name: CUDA_DEVICE_ORDER
+            value: "PCI_BUS_ID"
+          # Memory optimizations
+          - name: VLLM_ADDITIONAL_ARGS
+            value: "--gpu-memory-utilization 0.95 --max-model-len 8192 --enforce-eager"
+          - name: VLLM_ALL2ALL_BACKEND
+            value: deepep_high_throughput
+          - name: PYTORCH_CUDA_ALLOC_CONF
+            value: "expandable_segments:True"
+          # Essential NCCL configuration
+          - name: NCCL_IB_GID_INDEX
+            value: "3"
+          - name: NCCL_DEBUG
+            value: WARN
+          - name: NCCL_SOCKET_IFNAME
+            value: net1
+          - name: NCCL_IB_TIMEOUT
+            value: "100"
+          # NVSHMEM configuration - optimized for stability
+          - name: NVSHMEM_REMOTE_TRANSPORT
+            value: ibgda
+          - name: NVSHMEM_BOOTSTRAP_TWO_STAGE
+            value: "1"
+          - name: NVSHMEM_BOOTSTRAP_TIMEOUT
+            value: "300"
+          - name: NVSHMEM_BOOTSTRAP_UID_SOCK_IFNAME
+            value: net1
+          - name: NVSHMEM_IB_GID_INDEX
+            value: "3"
+          - name: NVSHMEM_USE_IBGDA
+            value: "1"
+          - name: NVSHMEM_ENABLE_NIC_PE_MAPPING
+            value: "1"
+          - name: NVSHMEM_IBGDA_SUPPORT
+            value: "1"
+          - name: NVSHMEM_IB_ENABLE_IBGDA
+            value: "1"
+          - name: NVSHMEM_IBGDA_NIC_HANDLER
+            value: "gpu"
+          # - name: NVSHMEM_IBGDA_RC_MAP_BY
+          #   value: "warp"
+          # - name: NVSHMEM_IBGDA_DCI_MAP_BY
+          #   value: "cta"
+          # - name: NVSHMEM_IBGDA_NUM_REQUESTS_IN_BATCH
+          #   value: "32"
```

Comment:
Would leaving a comment on this doc/sample help the reader to understand why those are left commented out?

Is this something we could set extra, and if so why? Or these are defaults?